velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -84,8 +84,8 @@ class DateTimeFunctionsTest : public functions::test::FunctionBaseTest {
   }
 
  public:
-  struct TimestampWithTimezone {
-    TimestampWithTimezone(int64_t milliSeconds, int16_t timezoneId)
+  struct TimestampWithTimezoneTest {
```

Comment:
adding the "Test" suffix to reduce confusion with the actual TimestampWithTimezone type